### PR TITLE
EOS-19817: cluster factory reset implementation: Config

### PR DESCRIPTION
**Pull request by akash2144**
Signed-off-by: Akash Dudhane <akash.a.dudhane@seagate.com>

## Problem Statement
<pre>
  <code>
  Story Ref (if any): EOS-19817
    HA: cluster factory reset implementation: Config
  </code>
</pre>
## Unit testing on RPM done
<pre>
  <code>
  Yes/No
  </code>
</pre>
## Problem Description
<pre>
  <code>
   For Config the following things are expected to be done in cleanup phase:
          1. Each node removes itself from the cluster
          2. Each node cleans /var/lib/pcsd directory (auth data)
          3. The last node destroys the cluster
  </code>
</pre>
## Solution
<pre>
  <code>
    1 & 3 points are already implemented.
    Cleaned /var/lib/pcsd directory
  </code>
</pre>
## Unit Test Cases
<pre>
  <code>
    Tested and confirmed that Cleanup command cleans/empty  "/var/lib/pcsd" directory.
  </code>
</pre>

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

**Review by ajay-paratmandali — APPROVED** [review]

File: ha/setup/ha_setup.py [hunk 1]
```diff
@@ -718,7 +718,12 @@ def remove_config_files(self):
         """
         Remove file created by ha.
         """
-        CleanupCmd.remove_file(const.CONFIG_DIR)
+        files = [const.CONFIG_DIR]
```
Comment (ajaysrivas) [inline ha/setup/ha_setup.py:721]: This is a directory. It should be named accordingly, should be named like config_dir.

Comment (akash2144) [reply ha/setup/ha_setup.py:721]: Here in these list i stored both directory and pcsd files so i named it as list.

**Review by indrajitzagade — APPROVED** [review]

File: ha/setup/ha_setup.py (hunk 1 above)
Comment (ajaysrivas) [reply ha/setup/ha_setup.py:721]: Ok

**Pull request merged**